src/geocoder/geocoder.ts

```diff
@@ -672,7 +669,9 @@ module powerbi.extensibility.geocoder {
                 return;
             }
 
-            GeocodeCallback = (data) => {
+            let guid =  "GeocodeCallback" + new Date().getTime() + (((1+Math.random())*0x10000)|0).toString(16).substring(1);;
```

Comment:
Please follow the following:
* Specify a type of the ```guid``` variable
* Use ```const``` instead of ```let```
* Add spaces around math operators
* Remove the second semicolon at the ending
* Use ES 2015 template string instead of concatenation